blueocean-pipeline-editor/.eslintrc

```diff
@@ -0,0 +1,58 @@
+{
+    "parserOptions": {
+        "ecmaFeatures": {
+            "jsx": true,
+            "modules": true
+        },
+        "sourceType": "module"
+    },
+    "env": {
+        "browser": true,
+        "es6": true,
+        "node": true
+    },
+    "parser": "babel-eslint",
+    "rules": {
+        "eol-last": 1,
+        "no-unused-vars": [0],
+        "max-len": [1, 160, 4],
+        "no-underscore-dangle": [0],
+        "object-shorthand": [0, "always"],
+        "quote-props": [0, "as-needed"],
+        "no-var": 0,
+        "prefer-const": 0,
+        "prefer-arrow-callback": 2,
+        "no-trailing-spaces": [0, {
+            "skipBlankLines": true
+        }],
+        "key-spacing": [2],
+        "semi": [2],
+        "no-extra-semi": 2,
+        "no-console": 0,
+        "prefer-template": 0,
+        "react/display-name": 0,
+        "react/jsx-boolean-value": 2,
+        "jsx-quotes": 2,
+        "react/jsx-no-undef": 2,
+        "react/jsx-sort-props": 0,
+        "react/jsx-uses-react": 2,
+        "react/jsx-uses-vars": 2,
+        "react/no-did-mount-set-state": 2,
+        "react/no-did-update-set-state": 2,
+        "react/no-multi-comp": 0,
+        "react/no-unknown-property": 2,
+        "react/prop-types": 2,
+        "react/react-in-jsx-scope": 2,
+        "react/self-closing-comp": 2,
+        "react/wrap-multilines": 2,
+        "quotes": [0, "single"],
+        "space-before-function-paren": [2, {
+            "anonymous": "always",
+            "named": "never"
+        }],
+        "strict": [2, "global"]
+    },
+    "plugins": [
+        "react"
+    ]
+}
```

Comment:
Ah sorry. I was just referring to the existence of another one of these files (containing a different set of rules). I never comment on "style" 😃  ... that's the advantage of a consistent set of automated rules ... people shouldn't have to comment.